src/Samples/Sample.AspNetCore/Controllers/VerifyController.cs

```diff
@@ -0,0 +1,115 @@
+                              IOptionsSnapshot<UrlsOptions> urlsAccessor,
+                              Cart cartService,
+                              ISwedbankPayClient swedbankPayClient, StoreDbContext context, SwedbankPayConnectionSettings swedbankPayConnectionSettings)
+        {
+            this.payeeInfoOptions = payeeInfoOptionsAccessor.Value;
+            this.urls = urlsAccessor.Value;
+            this.cartService = cartService;
+            this.swedbankPayClient = swedbankPayClient;
+            this.context = context;
+            this.swedbankPayConnectionSettings = swedbankPayConnectionSettings;
+        }
+
+        public IActionResult Index()
+        {
+            return View();
+        }
+
+
+        public IActionResult Privacy()
+        {
+            return View();
+        }
+         
+        public async Task<CardVerify> InitiateVerify()
+        {
+            try
+            {
+                var verifyRequest = new CardPaymentVerifyRequest(Operation.Verify, new CurrencyCode("SEK"),                                                                                 
+                                                                                   description = "Test Verification", userAgent = "useragent",
+                                                                                   CultureInfo.GetCultureInfo("sv-SE"),
+                                                                                   new Urls(this.urls.HostUrls, this.urls.VerificationListUrl,
+                                                                                            this.urls.TermsOfServiceUrl, this.urls.CancelUrl,
+                                                                                            this.urls.PaymentUrl, this.urls.CallbackUrl, this.urls.LogoUrl),
+                                                                                   new PayeeInfo(this.payeeInfoOptions.PayeeId,
+                                                                                                 this.payeeInfoOptions.PayeeReference),
+                                                                                   this.payeeInfoOptions.PayeeReference,
+                                                                                   generatePaymentToken = true,
+                                                                                   generateReccurenceToken = true)
+                                                                                   ;
+
+
+                var cardVerify = await this.swedbankPayClient.Payments.CardPayments.Verify(verifyRequest);
+                this.cartService.VerificationLink = cardVerify.VerifyResponse.Id.OriginalString;
+                this.cartService.Update();
+                //this.context.SaveChanges(true);
+
+                return cardVerify;
+                //var completeUrl = verifyResponse.VerifyResponse.Urls.CompleteUrl;
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=5742807)